packages/components/src/components/ViewerWithMenuBar/index.tsx

```diff
@@ -16,6 +17,8 @@ type Props = {
   playgroundSettings: PartialPlaygroundSettings;
   showMenu?: boolean;
   defaultTab?: ViewerTab;
+  randomizeSeed: (() => void) | undefined;
+  autorunMode?: boolean;
```

Comment:
This prop is only used in combination with `randomizeSeed` truthiness check.

It's also not very useful with `<SquiggleChart>` and `<SquiggleEditor>`.

So, send `randomizeSeed={undefined}` as a prop when autorun is on, and remove this prop?

Reply:
Btw, I also don't understand why you hide the re-roll button when autorun is disabled. "Run" in non-autorun mode doesn't re-roll, and it's still a useful action even in that mode.